samples/dapr-kafka/README.md

```diff
@@ -0,0 +1,5 @@
+## Run Kafka Docker Container locally with below command.
```

Comment:
Added readme here as it is being referred from [public doc](https://github.com/MicrosoftDocs/azure-docs-pr/pull/234679/files#diff-3df3b68d3dc1c60dc957a614990bf36bf274f223d8648f5fa1526919529e5fa0R195). So, having the instruction here will help user start the container.